### ISSUE TITLE
Add Zarafa attributes to LDAP index

### ISSUE THREAD

**Issue by jayme-github** [zarafa4ucs/zarafa4ucs#74]
We are missing some (all?) zarafa attributes in LDAP index.

At least we should add zarafaAccount to equality index as it is included in almost every query from zarafa.

**Comment by fbartels** [comment]
Good catch, that something I change on higher volume systems as well. According to https://doc.zarafa.com/trunk/Administrator_Manual/en-US/html-single/#_ldap_indices the following should do the trick:

```
dn: olcDatabase={1}hdb,cn=config
changetype: modify
add: olcDbIndex
olcDbIndex: cn eq
olcDbIndex: gidNumber eq
olcDbIndex: mail eq
olcDbIndex: memberUid eq
olcDbIndex: ou eq
olcDbIndex: uid eq
olcDbIndex: uidNumber eq
olcDbIndex: uniqueMember eq
olcDbIndex: zarafaAccount eq
olcDbIndex: zarafaAliases eq
olcDbIndex: zarafaViewPrivilege eq
```

**Comment by jayme-github** [comment]
As those "default" attributes are indexed already this strips down to the following on UCS systems:
```bash
eval "$(ucr shell ldap/index/eq)"
ucr set ldap/index/eq="$ldap_index_eq,zarafaAccount,zarafaAliases,zarafaViewPrivilege"
invoke-rc.d slapd stop
slapindex
invoke-rc.d slapd start
```

**Comment by edamrose** [comment]
Fixed in App Center Version Zarafa 7.1.14-51822
